<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgChat</class>
 <widget class="QDialog" name="DlgChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabChat">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>431</width>
     <height>471</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabGroup">
    <attribute name="title">
     <string>Group</string>
    </attribute>
    <widget class="QTableWidget" name="tblGroup">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>461</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbDel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>84</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSearch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Search:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAdd">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>0</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Chat Group:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leSearch">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leAddGrp">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbCancel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>250</y>
       <width>84</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbGrpAdd">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>0</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabChatSession">
    <attribute name="title">
     <string>Chat</string>
    </attribute>
    <widget class="QListWidget" name="lstChatMsg">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>301</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="edSendTxt">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>320</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSend">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>340</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click on this button to send the message.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lstChatMember">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>131</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbClear">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>320</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbXfrFile">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>340</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>File Transfer</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbBack">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click here to close.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
